raven/src/main/java/com/getsentry/raven/connection/BufferedConnection.java

```diff
@@ -158,6 +194,14 @@ public void run() {
                 while (events.hasNext() && !closed) {
                     Event event = events.next();
 
+                    long now = System.currentTimeMillis();
+                    long eventTime = event.getTimestamp().getTime();
+                    long age = now - eventTime;
+                    if (age < minAgeMillis) {
+                        logger.trace("Ignoring buffered event because it only " + age + "ms old.");
+                        return;
+                    }
```

Comment:
Makes sense, it'd be helpful to add a comment here explaining what's going on for future reference.